src/app/visit/visit.component.ts

```diff
@@ -21,15 +29,15 @@ export class VisitComponent implements OnInit {
         'type',
         'image',
     ];
-    constructor(
-        private storeService: StoreService,
-        private snackService: SnackService,
-    ) {}
-
+    dataSource;
```

Comment:
do we need this? we had a `store` variable for that, if you want to use `dataSource` please remove `store`